package exception;

/**
 * 使用当前类测试异常地抛出
 */
public class Person {
    public int age;

    public void setAge(int age) {
        this.age = age;
    }

    public int getAge() {
        return age;
    }
}
